doc/module.rst

```diff
@@ -181,18 +181,19 @@ please create the following XML block in your domain:
 
 .. note::
 
-   - Not using the new syntax above might cause QEMU to abort while starting the VM with
-     "``error: internal error: ... PCI: slot 1 function 0 not available for pcie-root-port, in use by ivshmem-plain``"
+   -  Using an incorrect syntax for your host configuration may cause QEMU to 
```

Comment:
The error doesn't show up just because you have "an incorrect syntax", but specifically if you use the legacy configuration when you should use the new one.

Reply:
This is what I mean by "incorrect syntax", is there a better way to put it?